effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -158,27 +158,32 @@ object JSNodeRunner extends Runner[String] {
     else Left("Cannot find nodejs. This is required to use the JavaScript backend.")
 
   /**
-   * Creates an executable `.js` file besides the given `.js` file ([[path]])
+   * Creates an executable `.mjs` file besides the given `.mjs` file ([[path]])
    * and then returns the absolute path of the created executable.
    */
   def build(path: String)(using C: Context): String =
     val out = C.config.outputPath().getAbsolutePath
-    val jsFilePath = (out / path).canonicalPath.escape
-    val jsFileName = path.unixPath.split("/").last
+    val mjsFilePath = (out / path).canonicalPath.escape
+    
     // create "executable" using shebang besides the .js file
-    val jsScript = s"require('./${jsFileName}').main()"
+    val mjsFileName = path.unixPath.split("/").last
+    
+    // NOTE: This is a hack since this file cannot use ES imports & exports
+    //       because it doesn't have the .mjs ending. Sigh.
+    //       Also, we add the 'file://' prefix to satisfy Windows.
+    val jsScript = s"import('file://${mjsFileName}').then(({main}) => { main(); })"
```

Comment:
I'll need to revisit this: the path should be relative (see #566).

Reply:
Let's just try using `file:filename` instead 🤔
```suggestion
    // NOTE: This is a hack since this file cannot use ES imports & exports
    //       because it doesn't have the .mjs ending. Sigh.
    //       Also, we add the 'file:' prefix to satisfy Windows.
    val jsScript = s"import('file:${mjsFileName}').then(({main}) => { main(); })"
```

Reply:
Uhoh.
> (node:8164) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_FILE_URL_PATH]: File URL path must be absolute

I'll need to think about this, worstcase we need to dynamically get the current file in JS...